packages/flutter_tools/test/integration.shard/flutter_attach_test.dart

```diff
@@ -204,9 +204,7 @@ void main() {
 
     testWithoutContext('enables Observatory on attach', () async {
       await flutterRun.run(withDebugger: true);
-      // Bail out if Observatory is still served by default in the VM.
-      // TODO(bkonyi): replace this with the following once Observatory is disabled in the VM:
-      // expect(await isObservatoryAvailable(), isFalse);
+      expect(await isObservatoryAvailable(), isFalse);
       if (await isObservatoryAvailable()) {
```

Comment:
This line can be removed now that the expectation is checking the result.

Reply:
Done.